src/gval/comparison/agreement.py

```diff
@@ -226,7 +194,6 @@ def _manage_information_loss(agreement_map, crs, nodata, encode_nodata):
 
     agreement_map = _manage_information_loss(
         agreement_map=agreement_map,
```

Comment:
Do we not need the CRS anymore? I thought that was an issue with xr.apply_ufunc?

Reply:
The crs argument was not being used in that method.  Instead the following was in place:
`agreement_map.rio.set_crs(candidate_map.rio.crs, inplace=True)  `
Thus I removed the argument.  If this method gets abstracted away from agreement it should be re-added.  I could change it back and use crs instead if that is preferred.

Reply:
This has been adjusted